assets/i18n/vi.json

```diff
@@ -175,6 +175,18 @@
   "I18N_PLAYER_RATE_EXPLORATION": "Bạn đã học được điều gì mới mẻ? Hãy đánh giá cuộc hành trình này",
   "I18N_PLAYER_RATINGS_TOOLTIP": "Điểm đánh giá",
   "I18N_PLAYER_RECOMMEND_EXPLORATIONS": "Đề nghị kế tiếp:",
+  "I18N_PLAYER_REPORT_MODAL_BODY_AD": "Sai lệch nội dung, thư rác hay quảng cáo",
```

Comment:
@TruongKim @MinhHuong Please could you take a look at this, and let us know if any changes are needed? Thanks!